python/beeai_framework/workflows/agent/agent.py

```diff
@@ -34,16 +33,16 @@
 
 
 class AgentWorkflowInput(BaseModel):
-    prompt: str | None = None
-    context: str | None = None
+    input: str
+    backstory: str | None = None
```

Comment:
I don't think the breaking change is necessary in something we will soon remove.

Reply:
Undone.